README.rst

```diff
@@ -2,17 +2,16 @@ intel-iot-refkit
 ################
 
 This README file contains information on setting up, building, and booting
-Iot Reference OS Kit for Intel(r) Architecture project.
+IoT Reference OS Kit for Intel(r) Architecture project.
 
```

Comment:
Should we add a "the" prior to "IoT Reference OS..." so the sentence would read: "...building, and booting the IoT Reference OS Kit for Intel(r)..."

Is it appropriate to have a comma followed by "and" ("...building, and booting...")